maps/nerva/nerva.dm

```diff
@@ -65,6 +65,8 @@
 	#include "../away/stations/pirate/pirate_station.dm"
 	#include "../away/voxship/voxship.dm"
 	#include "../away/glloyd_jungle/glloyd_jungle.dm"
+	#include "../away/forest/forest.dm"
+	#include "../away/forest/forest_jobs.dm"
```

Comment:
Shouldn't this be included inside of `forest.dm` itself?

Reply:
Probably

Reply:
it should, yeah. you also committed your map backups, which should be removed.